hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -1491,6 +1491,11 @@ hazelcast:
           indexes:
             - attributes:
               - "name"
+      # In an actual config, only one of {partition-strategy, partition-attributes} should be present
+      partition-strategy: "com.example.Strategy"
+      partition-attributes:
+        - name: "attributeName1"
+        - name: "attributeName2"
```

Comment:
One thing I don't understand is why yaml config is different than xml. What I mean is why do we have "name" in yaml but "attribute" in xml? Why do they differ?

Reply:
XML node content is used in XML in place of name, but for YAML an explicit property is required. If you feel strongly about this I can change the `name` to attribute, but I think it would be less intuitive.

Reply:
TBH, I don't have a strong preference. I just like when xml and yaml has identical node structure because it is easier for me to wrap my head around it. Given this is a kind of nitpicking, treat this as resolved if you will.